src/app/core/services/repo-creator.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { flatMap } from 'rxjs/operators';
+import { Observable, of, pipe, UnaryFunction } from 'rxjs';
+import { GithubService } from './github.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class RepoCreatorService {
+  constructor(
+    private githubService: GithubService
+  ) {}
+
+  /**
+   * Checks if the necessary repository has been created.
```

Comment:
```suggestion
   * Checks if the specified repository has been created.
```
Let's add `@param` tags to explain the parameters (phaseOwner is the organisation / user holding the repo, and phaseRepo is the name of the repo)

Reply:
Alright, will add this in as well!